include/atomicassets.hpp

```diff
@@ -11,6 +11,7 @@ using namespace atomicdata;
 
 
 static constexpr double MAX_MARKET_FEE = 0.15;
+static constexpr uint32_t AUTHOR_SWAP_TIME_DELTA = 604800; // 1 week, valid for 3 weeks
```

Comment:
Just personal preference, but I would prefer if this was written out as 60 * 60 * 24 * 7 so that its immediately clear what this means and that it's correct.